packages/react-reconciler/README.md

```diff
@@ -318,6 +318,26 @@ Same as `unhideInstance`, but for nodes created by `createTextInstance`.
 
 This method should mutate the `container` root node and remove all children from it.
 
+#### `maySuspendCommit(type, props)`
+
+This method should determine if this combination of type and props may need to suspend.
+
+#### `preloadInstance(type, props)`
+
+This method should kick off any preloading of resources required. Return `true` to indicate it's already loaded or `false` to trigger a fallback.
```

Comment:
Called during render
```suggestion
This method may be called during render if the Host Component type and props might suspend a commit. It can be used to initiate any work that might shorten the duration of a suspended commit.
```